Mail/Views/Thread/MessageListView.swift

```diff
@@ -46,16 +55,21 @@ struct MessageListView: View {
                 computeExpansion(from: messages.toArray())
 
                 guard messages.count > 1,
-                      let firstExpanded = firstExpanded() else { return }
+                      let firstExpandedUid = firstExpanded()?.uid else {
+                    return
+                }
+
+                // TODO: listen for last message `.isFulyLoaded` to be exact
                 DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
                     withAnimation {
-                        proxy.scrollTo(firstExpanded.uid, anchor: .top)
+                        proxy.scrollTo(firstExpandedUid, anchor: .top)
                     }
                 }
             }
             .onChange(of: messages) { newValue in
                 computeExpansion(from: newValue.toArray())
             }
+            .id(self.id)
```

Comment:
It works without this on iPhone as it is a different screen. It did not work on iPad.

Reply:
Or we can just remove the animation on iPad.

Reply:
Discussed